app/base/handlers.js

```diff
@@ -6,11 +6,15 @@ const raven = require('raven');
 
 
 exports.home = (req, res, next) => {
-  if (req.user.is_superuser) {
-    res.render('home/superuser.html');
-  } else {
-    res.render('home/user.html');
-  }
+  User.get(req.user.auth0_id)
+    .then((user) => {
+      res.render('base/home.html', {
+        signedInUser: true,
+        user,
+        isSuperuser: req.user.is_superuser,
```

Comment:
The template has a `current_user` variable already available, so instead of passing `isSuperuser`, you could just use `current_user.is_superuser`. Also, you're passing the current user to the template as `user` here, so you could also use `user.is_superuser`